packages/frontend/src/pages/bridges-risk/BridgesRiskView.tsx

```diff
@@ -79,13 +88,22 @@ export function BridgesRiskView({ items }: BridgesRiskViewProps) {
   ]
 
   const rows: RowConfig<BridgesRiskViewEntry> = {
-    getProps: (entry) =>
-      entry.type === 'bridge'
-        ? {}
-        : {
-            ['data-combined-only']: true,
-            className: 'hidden',
-          },
+    getProps: (entry) => {
+      const result: Record<string, string | boolean> = {
+        className: '',
+      }
+
+      if (entry.isVerified === false) {
```

Comment:
This `if` statement is unnecessary - you're checking this in `getRowVerificationClassNames` already.